packages/server/WebServer/graphql.js

```diff
@@ -29,8 +29,13 @@ const ttyLink = (text, path, port, version) => {
   }
 };
 
-module.exports = function createGraphQLMiddleware(server, { apiPath, graphiqlPath, port }) {
+module.exports = function createGraphQLMiddleware(
+  server,
+  { apiPath, graphiqlPath, port },
+  { public, audiences }
```

Comment:
`public` 💥 

`isPublic`? It would then match the `isUnique` / `isRequired` API used elsewhere.

Reply:
Yep, `isPublic` is a much better idea 😅